Configuration/PasswordSequenceConfiguration.cs

```diff
@@ -13,6 +16,23 @@ public PasswordSequenceConfiguration()
 
             Sequence = new List<SequenceItem>();
         }
+        
+        /* this constructor is for creating an empty configuration if reading
+         * the config file fails somehow; the argument is unused but I don't
+         * know a better way to do it
+         */
+        public PasswordSequenceConfiguration(bool throwaway)
+        {
+            XmlNamespace = new XmlSerializerNamespaces();
+            XmlNamespace.Add(string.Empty, "http://quasivirtuel.com/PasswordSequenceConfiguration.xsd");
+
+            Sequence = new List<SequenceItem>();

```

Comment:
Might as well just fallback to the default constructor and remove anything above this line.
...using `public PasswordSequenceConfiguration(bool throwaway):this()`